packages/firebase/README.md

```diff
@@ -0,0 +1,68 @@
+<p align="center">
```

Comment:
Could you make this README resemble the typeorm one?

Reply:
Yeah no problem :+1: